Marlin/src/module/probe.cpp

```diff
@@ -814,4 +816,85 @@ float Probe::probe_at_point(const_float_t rx, const_float_t ry, const ProbePtRai
 
 #endif // HAS_Z_SERVO_PROBE
 
+#if ENABLED(SENSORLESS_PROBING)
```

Comment:
consider also SENSORLESS_HOMING

Reply:
The same as the previous .

Reply:
> consider also SENSORLESS_HOMING

Done.

Reply:
Hello.
I am unable to understand why this code fails the checks.
![image](https://user-images.githubusercontent.com/83166168/122816914-92f0fa80-d2d7-11eb-87e4-0ca655ef3b53.png)

It is identical to the one in "probe.h".
6 commits trying to fix it, I give up. I need some C ++ classes.